code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -284,6 +284,8 @@
 	desc = "A long ranged weapon that does significant damage. No, you can't quickscope."
 	icon_state = "sniper"
 	inhand_icon_state = "sniper"
+	worn_icon_state = null
+	worn_icon = 'icons/mob/clothing/back.dmi'
```

Comment:
build_worn_icon

Reply:
Reverted. Had the worn_icon set to make the other slots use a working icon. I'll add it to the list of missing textures.